docs/og-analytics/worked-out-examples/worked-out-example-stir-futures-option-normal-moneyness.rst

```diff
@@ -0,0 +1,112 @@
+
+    NormalSTIRFuturesExpSimpleMoneynessProviderDiscount NORMAL_MULTICURVES = 
+      new NormalSTIRFuturesExpSimpleMoneynessProviderDiscount(MULTICURVES, VOL_SURFACE_SIMPLEMONEY, GBPLIBOR3M, false);
+
+Here the simple moneyness is given by the underlying futures price minus strike price for **false** input. 
+
+Instrument
+----------
+First we create an underlying STIR futures security with data stored in the example::
+
+    InterestRateFutureSecurityDefinition ERU2_DEFINITION = 
+      new InterestRateFutureSecurityDefinition(LAST_TRADING_DATE, EURIBOR3M, NOTIONAL, FUTURE_FACTOR, NAME, TARGET);
+
+Then we crate an option on the STIR futures::
+
+    InterestRateFutureOptionMarginSecurityDefinition OPTION_ERU2_DEFINITION = 
+      new InterestRateFutureOptionMarginSecurityDefinition(ERU2_DEFINITION, EXPIRATION_DATE, STRIKE, IS_CALL);
+
+We create a transaction on the above security with data stored in the file::
+
+    InterestRateFutureOptionMarginTransactionDefinition TRANSACTION_1_DEFINITION = 
+      new InterestRateFutureOptionMarginTransactionDefinition(OPTION_ERU2_DEFINITION, QUANTITY, TRADE_DATE_1, TRADE_PRICE);
+
+The transaction is the purchase/sell of a given quantity on a given date for a given price.
+ 
+This is the definition version of the transaction, i.e. it contains all the dates and data relevant to the trade.
+
+The definition version is converted to the *derivative* version::
+
+    InterestRateFutureOptionMarginTransaction TRANSACTION_1 = TRANSACTION_1_DEFINITION.toDerivative(REFERENCE_DATE, MARGIN_PRICE);
+
+In this transformation, the dates are replaced by time between the valuation date and the different cash flow dates. Moreover the last priced used in the margining is stored. This is used to compute the present value which is the difference between the current price and the last margin price.
+
+
+In this examples all the computation are done with a *projection and discounting* model, that we simply call *discounting*. No convexity adjustment for the daily margining is computed in this implementation. Another worked out example deal with convexity adjustment; **STIRFuturesTransactionHullWhiteMethodE2ETest**.
+
+
+Present value
+-------------
+
+The first output we produce is the *present value* of the instrument. The present value is computed as the current price minus the last margin price multiplied by the notional, the accrual factor and the quantity.
+
+The code to compute the present value simply read as::
+
+    private static final PresentValueNormalSTIRFuturesCalculator PVNFC = PresentValueNormalSTIRFuturesCalculator.getInstance();
+    MultipleCurrencyAmount pv = TRANSACTION_1.accept(PVNFC, NORMAL_MULTICURVES);
+
+The present value calculator is used through the visitor pattern. The result of the present value computation is a **MultipleCurrencyAmount** involving the information about the currency. The present value calculator is further described in :doc:`../PresentValueDiscountingCalculator`.
+
```

Comment:
The links do not work properly in GitHub. We have to review how to have those links working properly.